src/MOI_wrapper.jl

```diff
@@ -420,6 +423,64 @@ function MOI.get(::Optimizer, ::MOI.ListOfOptimizerAttributesSet)
     return attributes
 end
 
+function MOI.compute_conflict!(o::Optimizer)
+    if o.conflict_status != MOI.COMPUTE_CONFLICT_NOT_CALLED
+        error("Conflict computation is destructive for the model and cannot be called twice.")
+    end
+    # free the transformed problem first
+    if LibSCIP.SCIPgetStage(o) != LibSCIP.SCIP_STAGE_PROBLEM
+        @SCIP_CALL LibSCIP.SCIPfreeTransform(o)
+    end
+    # we need names for all constraints
+    for (F, S) in MOI.get(o, MOI.ListOfConstraintTypesPresent())
+        for (idx, c_index) in enumerate(MOI.get(o, MOI.ListOfConstraintIndices{F,S}()))
+            if MOI.get(o, MOI.ConstraintName(), c_index) == ""
+                cons_ptr = cons(o, c_index)
+                handler_name = unsafe_string(SCIPconshdlrGetName(SCIPconsGetHdlr(cons_ptr)))
+                cons_name = "$(handler_name)_moi_$(idx)"
+                MOI.set(o, MOI.ConstraintName(), c_index, cons_name)
+            end
+        end
+    end
+    success = Ref{LibSCIP.SCIP_Bool}(333)
```

Comment:
Why `333`? Perhaps `SCIP.FALSE`?

Reply:
yeah this was as a flag value to indicate that it hasn't been touched